src/main/resources/bin/mta-cli

```diff
@@ -220,13 +220,11 @@ fi
 MODULES=""
 JAVAVER=`"$JAVACMD" -version 2>&1`
 case $JAVAVER in
-"11."*)
+*"11"*)
```

Comment:
Just for adding a not about this change. It seems our master branch content is wrong and after an internal chat with @mrizzi we noticed the condition code needed to be fixed. I've tested and verified this condition locally.